source/docs/contributing/style-guide.rst

```diff
@@ -30,7 +30,7 @@ All text content should be on the same line, if you need readability, use the wo
 
 Use the following case for these terms:
 
-- roboRIO (not RoboRIO, roboRio, or RoboRio)
+- roboRIO (not roboRIO, roboRio, or RoboRio)
```

Comment:
🙂 This is the only place where we want it.
```suggestion
- roboRIO (not RoboRIO, roboRio, or RoboRio)
```

Reply:
Whoops. Used a script to do it :)